client/express-checkout/blocks/index.js

```diff
@@ -39,7 +39,7 @@ export const expressCheckoutElementApplePay = ( api ) => ( {
 			return false;
 		}
 
-		return checkPaymentMethodIsAvailable( 'applePay', cart );
```

Comment:
Providing `api` as an additional argument, so this function doesn't have to create a new instance of it.